CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 5.0.1
+
+* Add a zones parameter
+* Make $localzonepath and $defaultzonepath required parameters
```

Comment:
I wonder if it would be clearer if we said ```disallow undef values in favor of unmanaged``` or if this is clear enough already.

Reply:
rephrased it a bit